src/index.js

```diff
@@ -1,7 +1,160 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
+import { render } from "react-dom";
+import React from "react";
+import L from "leaflet";
+import borderData from "./border.js";
+import countyData from "./vtcountyborder";
+import leafletPip from "@mapbox/leaflet-pip";
 
-ReactDOM.render(
-  <h1>Hello World!</h1>,
-  document.getElementById('root')
-)
\ No newline at end of file
+
+const mapStyle = {
+  width: "100%",
+  height: "400px",
+  pointerEvents: "none"
+};
+
+function getRandomArbitrary(min, max) {
```

Comment:
Everything between lines 14 and 45 should be moved inside of your Map class. To refactor it further, consider using state values and bound functions to assign numerical values important to your game instead of using global variables and separate functions.

Reply:
Although your code functions properly, applying these refactors will increase the readability of your code